gp-templates/translations.php

```diff
@@ -25,49 +25,13 @@
 		<?php echo gp_link_get( gp_url( '/glossaries/-new', array( 'translation_set_id' => $translation_set->id ) ), __( 'Create Glossary', 'glotpress' ), array('class'=>'glossary-link') ); ?>
 	<?php endif; ?>
 </h2>
-<?php if ( $can_approve ): ?>
-<form id="bulk-actions-toolbar" class="filters-toolbar bulk-actions" action="<?php echo $bulk_action; ?>" method="post">
-	<div>
-	<select name="bulk[action]" id="bulk-action">
-		<option value="" selected="selected"><?php _e( 'Bulk Actions', 'glotpress' ); ?></option>
-		<option value="approve"><?php _e( 'Approve', 'glotpress' ); ?></option>
-		<option value="reject"><?php _e( 'Reject', 'glotpress' ); ?></option>
-		<option value="fuzzy"><?php _e( 'Fuzzy', 'glotpress' ); ?></option>
-	<?php if( $can_write ) : ?>
-		<option value="set-priority" class="hide-if-no-js"><?php _e( 'Set Priority', 'glotpress' ); ?></option>
-	<?php endif; ?>
-		<?php
+<?php
+if ( $can_approve ) {
+	gp_translations_bulk_actions_toolbar( $bulk_action, $can_write, 'top' );
+}
 
-		/**
-		 * Fires inside the bulk action menu for translation sets.
-		 *
-		 * Printing out option elements here will add those to the translation
-		 * bulk options drop down menu.
-		 *
-		 * @since 1.0.0
-		 *
-		 * @param GP_Translation_Set $set The translation set.
-		 */
-		do_action( 'gp_translation_set_bulk_action', $translation_set ); ?>
-	</select>
-	<?php if( $can_write ) : ?>
-	<select name="bulk[priority]" id="bulk-priority" class="hidden">
-	<?php
-	foreach( GP::$original->get_static( 'priorities' ) as $value => $label ) {
-		$selected = $value == 'normal' ? " selected='selected'" : '';
-		echo "\t<option value='" . esc_attr( $value ) . "' $selected>" . esc_html( $label ) . "</option>\n";
-	}
-	?>
-	</select>
-	<?php endif; ?>
-	<input type="hidden" name="bulk[redirect_to]" value="<?php echo esc_attr(gp_url_current()); ?>" id="bulk[redirect_to]" />
-	<input type="hidden" name="bulk[row-ids]" value="" id="bulk[row-ids]" />
-	<input type="submit" class="button" value="<?php esc_attr_e( 'Apply', 'glotpress' ); ?>" />
-	</div>
-	<?php gp_route_nonce_field( 'bulk-actions' ); ?>
-</form>
-<?php endif; ?>
-<?php echo gp_pagination( $page, $per_page, $total_translations_count ); ?>
+echo gp_pagination( $page, $per_page, $total_translations_count ); // WPCS: XSS Ok.
```

Comment:
If `gp_pagination()` is safe we should add to `customAutoEscapedFunctions` in `phpcs.ruleset.xml`